packages/cli-platform-android/src/config/findManifest.ts

```diff
@@ -23,7 +23,16 @@ export default function findManifest(folder: string) {
       '**/src/androidTest/**',
       '**/src/test/**',
     ],
-  })[0];
+  });
+  if (manifestPaths.length > 1) {
+    // if we have more than one manifest, pick the one in the main folder if present
+    const mainManifest = manifestPaths.filter((manifestPath) =>
+      manifestPath.includes('src/main/'),
+    );
+    if (mainManifest.length === 1) {
+      manifestPaths = mainManifest;
```

Comment:
nit: how about not mutating `manifestPaths` and extracting the `mainManifest` to a variable to be used later? If we want to make sure only one main manifest is picked, we could use `.find` instead of `.filter`.

Reply:
Ok, let's stick to the current behavior. Unit tests job on Node 20 is flaky, try not to bother it :D